glossary.dd

```diff
@@ -81,6 +81,14 @@ $(DL
 	maximize the portability of code.
 	)
 
+	$(DT $(LNAME2 lvalue, lvalue))
+	$(DD A lvalue is the counterpart to a rvalue. A lvalue is an abstract term
```

Comment:
An lvalue does read better but is it correct to write that? I've no idea. Has this discussion taken place before? Agreed the counterpart sentence is better appearing at the end.